apple-codesign/src/yubikey.rs

```diff
@@ -156,7 +159,7 @@ fn attempt_authenticated_operation<T>(
 /// Represents a connection to a yubikey device.
 pub struct YubiKey {
     yk: Arc<Mutex<RawYubiKey>>,
-    pin_callback: Option<PinCallback>,
```

Comment:
While re-reading this PR, I just realized that this `Rc` makes `YubiKey` `!Send`. Should probably turn this into an `Arc` so we keep the `Send` bound and avoid introducing a breaking change to the lib.